qiskit_optimization/translators/model_translator.py

```diff
@@ -0,0 +1,43 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Abstract class for optimization model translators"""
+
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class ModelTranslator(ABC):
+    """Translator between an optimization model and a quadratic program
+    """
+
+    @abstractmethod
+    def qp_to_model(self, quadratic_program: Any) -> Any:
```

Comment:
Same as above, why `quadratic_program: Any`, but not `quadratic_program: QuadraticProgram` ?